tmp/wrapper_generated.py

```diff
@@ -0,0 +1,284 @@
+
```

Comment:
This file is also just included to see the results of the codegen. It would not be part of a real PR. It can be used as a standalone runner and to experiment with the code as it is really pretty much self-contained. It's generated within wrapper.py